src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerGenotypingEngine.java

```diff
@@ -672,24 +666,6 @@ protected static void cleanUpSymbolicUnassembledEvents( final List<Haplotype> ha
         return new EventMap(haplotype, ref, refLoc, sourceNameToAdd);
     }
 
-    protected static final class Event {
```

Comment:
When I figured out that `Event` was just a wrapper for `VariantContext` with a different `hashCode` I wanted to bang my forehead against a wall.